synthtool/gcp/gapic_bazel.py

```diff
@@ -47,6 +47,7 @@ def __init__(self):
         self._googleapis_private = None
         self._googleapis_discovery = None
         self._discovery_artifact_manager = None
+        self._clean_build = False
```

Comment:
Probably prefer this to be set via the constructor args (or keyword arg to `_generate_code()`). `_variables` are meant to signal private so you shouldn't configure this via `gapic._clean_build = True`.

Reply:
Done.